### PR TITLE
[Ultica] Corpses and some misc items

### PR DESCRIPTION
**Pull request by MrAsbestos**
Adds the zombie soldier and headless zombie corpse with it's wielded overlays.

Also adds the correct sprites for the baseball and hockey puck and adds the MRE.

### CODE REVIEW

**Comment by MrAsbestos** [comment]
Still working on it, do not merge please

**Comment by int-ua** [comment]
> Still working on it, do not merge please

Please convert it to Draft until ready https://github.com/CleverRaven/Cataclysm-DDA/blob/master/.github/CONTRIBUTING.md#drafts

File: gfx/UltimateCataclysm/pngs_tall_32x64/monsters/mon_zombie_headless_variants/mon_zombie_headless.json [hunk 1]
```diff
@@ -1,10 +1,14 @@
...
     "fg": [
       { "weight": 100, "sprite": "zombie_headless_f" },
       { "weight": 100, "sprite": "zombie_headless_m" }
     ],
   "bg": "shadow_32x64_normal"
-}
+  },
+  {
+    "id": "corpse_mon_zombie_brainless",
+    "fg": "corpse_mon_zombie_brainless",
```
Comment (int-ua) [inline gfx/UltimateCataclysm/pngs_tall_32x64/monsters/mon_zombie_headless_variants/mon_zombie_headless.json:12]:
```suggestion
    "fg": "corpse_mon_zombie_brainless"
```

Comment (int-ua) [reply gfx/UltimateCataclysm/pngs_tall_32x64/monsters/mon_zombie_headless_variants/mon_zombie_headless.json:12]: @MrAsbestos `bg` should not be required now, can you please try removing `bg` and the comma? Yes, it works this way, I just would like to show and confirm that it will work without bg

File: gfx/UltimateCataclysm/pngs_tall_32x64/overlay/wielded/corpses/corpses.json [hunk 2]
```diff
@@ -10,13 +10,33 @@
...
...ale_wielded_corpse_mon_zombie_child_fungus", "overlay_male_wielded_corpse_mon_zombie_swimmer", "overlay_male_wielded_corpse_mon_zombie_mancroc", "overlay_male_wielded_corpse_mon_zombie_soldier", "overlay_male_wielded_corpse_mon_zombie_skull", "overlay_male_wielded_corpse_mon_zombie_brainless", "overlay_male_wielded_corpse_mon_zombie_survivor", "overlay_male_wielded_corpse_mon_zombie_brute_shocker", "overlay_male_wielded_corpse_mon_zombie_soldier_acid_1", "overlay_male_wielded_corpse_mon_zombie_soldier_blackops_2", "overlay_male_wielded_corpse_mon_zombie_soldier_blackops_1", "overlay_male_wielded_corpse_mon_zombie_soldier_acid_2", "overlay_male_wielded_corpse_mon_zombie_shriekling", "overlay_male_wielded_corpse_mon_zombie_ears", "overlay_male_wielded_corpse_mon_zombie_nullfield", "overlay_male_wielded_corpse_mon_zombie_waif", "overlay_male_wielded_corpse_mon_zombie_sproglodyte", "overlay_male_wielded_corpse_mon_zombie_creepy", "overlay_male_wielded_corpse_mon_zombie_anklebiter", "overlay_male_wielded_corpse_mon_zombie_bio_op", "overlay_male_wielded_corpse_mon_zombie_armored", "overlay_male_wielded_corpse_mon_zombie_prisoner", "overlay_male_wielded_corpse_mon_zombie_military_pilot"],
```
Comment (int-ua) [inline gfx/UltimateCataclysm/pngs_tall_32x64/overlay/wielded/corpses/corpses.json:13]: oh, please split these long lines while you are at it, one id per line

**Comment by MrAsbestos** [comment]
Ready to merge!
The only thing that it's not working is wielded zombie corpses, but that's a CDDA problem, not the tileset.

**Comment by Fris0uman** [comment]
Should we add the wielded sprites if they're not going to be displayed anyway? NO reason to "bloat" the tilesheets with sprites that will never appear. Maybe they should be put on standby in a folder that is not used by the compose script and we'll add them to the set proper once the game can actually display them

File: gfx/UltimateCataclysm/pngs_tall_32x64/monsters/mon_zombie_headless_variants/mon_zombie_headless.json [hunk 3]
```diff
@@ -1,10 +1,15 @@
...
     ],
   "bg": "shadow_32x64_normal"
-}
+  },
+  {
+    "id": "corpse_mon_zombie_brainless",
+    "fg": "corpse_mon_zombie_brainless"
+  }
 ]
+
```
Comment (Fris0uman) [inline gfx/UltimateCataclysm/pngs_tall_32x64/monsters/mon_zombie_headless_variants/mon_zombie_headless.json:15]:
```suggestion
```
try to not add unecessary lines, you can the dda linter http://dev.narc.ro/cataclysm/format.html to format the json nicely

File: gfx/UltimateCataclysm/pngs_tall_32x64/monsters/mon_zombie_soldier_variants/mon_zombie_soldier.json [hunk 4]
```diff
@@ -8,5 +8,15 @@
...
+    "id": [
+      "corpse_mon_zombie_soldier",
+      "corpse_mon_zombie_military_pilot"
+    ],
+    "fg": "corpse_mon_zombie_soldier"
   }
 ]
+
+
+
```
Comment (Fris0uman) [inline gfx/UltimateCataclysm/pngs_tall_32x64/monsters/mon_zombie_soldier_variants/mon_zombie_soldier.json:22]:
```suggestion
```
same

**Comment by MrAsbestos** [comment]
I think I am gonna remove the wielded sprites for the corpses for now

File: gfx/UltimateCataclysm/pngs_tall_32x64/monsters/mon_zombie_headless_variants/mon_zombie_headless.json [hunk 5]
```diff
@@ -1,10 +1,11 @@
...
-    "fg": [
-      { "weight": 100, "sprite": "zombie_headless_f" },
-      { "weight": 100, "sprite": "zombie_headless_m" }
-    ],
-  "bg": "shadow_32x64_normal"
-}
-]
+  {
+    "id": "mon_zombie_brainless",
+    "fg": [ { "weight": 100, "sprite": "zombie_headless_f" }, { "weight": 100, "sprite": "zombie_headless_m" } ],
```
Comment (int-ua) [inline gfx/UltimateCataclysm/pngs_tall_32x64/monsters/mon_zombie_headless_variants/mon_zombie_headless.json:4]: I guess it was changed automatically by some linter? Can you please revert joining this line as it is less readable now?

File: gfx/UltimateCataclysm/pngs_tall_32x64/overlay/wielded/corpses/corpses.json [hunk 6]
```diff
@@ -19,4 +19,4 @@
     "fg": "mon_zombie_f",
     "bg": ""
   }
-]
+]
```
Comment (int-ua) [inline gfx/UltimateCataclysm/pngs_tall_32x64/overlay/wielded/corpses/corpses.json:22]:
```suggestion
]

```

Comment (int-ua) [inline gfx/UltimateCataclysm/pngs_tall_32x64/overlay/wielded/corpses/corpses.json:22]: Please revert this change

**Comment by MrAsbestos** [comment]
wait a minute...
I think this whole time my commetns weren't visible....
lol
